index.html

```diff
@@ -207,21 +207,32 @@ <h5>Get started</h5>
       </p>
     </div>
 
-    <ul class="form-name-box">
-      <li class="form-name-gray">Your name</li>
-      <li class="fill-name">Anastasiia</li>
-    </ul>
-
-    <ul class="form-email-box">
-      <li class="form-email-gray">Your email</li>
-      <li class="fill-email">|</li>
-    </ul>
-    <ul class="message-box">
-      <li class="fill-messagge">Enter your message here...</li>
-    </ul>
+    <form action="https://formspree.io/f/myyobvpo" method="Post" class="form-name-box">
+      <div class="form-name-gray"><label for="name">Your name</label></div>
+      <div class="fill-name">
+        <input type="text" name="name" id="name" value="" placeholder="Your name" maxlength="30" required>
+      </div>
+
+    </form>
+
+    <form action="https://formspree.io/f/myyobvpo" method="POST" class="form-email-box">
+      <div class="form-email-gray"><label for="text">e-mail</label></div>
+      <div class="fill-email"><input type="email" name="e-mail" id="text" value="" placeholder="|" maxlength="30"
+          required>
+      </div>
+    </form>
+
+    <form class="message-box">
+      <div class="fill-messagge"><label for="text"></label>
+        <textarea name="name" id="form" cols="36" rows="7" maxlength="500" wrap="soft"
+          placeholder="Enter your message here..." required></textarea>
+
+      </div>
+    </form>
     <div class="start-collab-button-2">
-      <button>Start collaboration</button>
+      <button type="submit">Start collaboration</button>
     </div>
+
```

Comment:
- [ ] As suggested by the previous reviewer kindly use the form tag to wrap the entire form from [here](https://www.w3schools.com/html/html_forms.asp). Make sure to use the form tag only once and not repeat it for each input.
- [ ] Kindly make sure to submit form data to Formspree as per the project requirement. Right now your form is not working. In order to make it work use the form tag only once and the form spree link in it.